api/incubator/src/test/java/io/opentelemetry/api/incubator/config/InstrumentationConfigUtilTest.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.incubator.config;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.collect.ImmutableMap;
+import io.opentelemetry.api.incubator.config.internal.YamlStructuredConfigProperties;
+import io.opentelemetry.sdk.extension.incubator.fileconfig.FileConfigProvider;
+import io.opentelemetry.sdk.extension.incubator.fileconfig.FileConfiguration;
+import io.opentelemetry.sdk.extension.incubator.fileconfig.internal.model.OpenTelemetryConfiguration;
+import java.io.ByteArrayInputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import org.junit.jupiter.api.Test;
+
+class InstrumentationConfigUtilTest {
+
+  /**
+   * See <a
+   * href="https://github.com/open-telemetry/opentelemetry-configuration/blob/main/examples/kitchen-sink.yaml">kitchen-sink.yaml</a>.
+   */
+  private static final String kitchenSinkInstrumentationConfig =
+      "instrumentation:\n"
```

Comment:
I assume we aren't using a different source level for tests and main, and that's why we aren't using multi-line literals? If this is something we'd like to change later, totally suggest openrewrite and then the test source can be nifty at the cost of the build (not runtime) floor JDK being one that supports it.

Reply:
Yes we use same source level for tests and main. I think when we discussed the possibility of a higher level for tests we rejected the idea with something to the effect of "high cognitive load to switch between multiple source levels in the same project". Personally, I've used different source levels in test and mains and liked it. Plus it allows me to stay more practiced with modern java syntax.